tests/builder.rs

```diff
@@ -67,8 +68,11 @@ impl BundleOrchardFlavor for OrchardZSA {
     const SPENDS_DISABLED_FLAGS: Flags = Flags::SPENDS_DISABLED_WITH_ZSA;
 }
 
-fn bundle_chain<FL: BundleOrchardFlavor>() {
-    let mut rng = OsRng;
+fn bundle_chain<FL: BundleOrchardFlavor>(
+    expected_orchard_digest_1: [u8; 32],
+    expected_orchard_digest_2: [u8; 32],
+) {
```

Comment:
It is more natural to return the resulted digests and assert externally.

(optional, since we have many asserts inside)